### PR TITLE
climate setting schedules search and filter

### PR DESCRIPTION
**Pull request by Andrewjeska**

### CODE REVIEW

File: src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleTable.stories.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { useToggle } from 'lib/ui/use-toggle.js'
 import { ClimateSettingScheduleTable } from './ClimateSettingScheduleTable.js'
 
 const meta: Meta<typeof ClimateSettingScheduleTable> = {
-  title: 'Example/ClimateSettingScheduleTable',
+  title: 'Example/ClimateSettingScheduleTable (Coming Soon)',
```
Comment (Andrewjeska) [inline src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleTable.stories.tsx:9]: We chatted about adding this

File: src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleTable.element.ts [hunk 2]
```diff
@@ -6,7 +6,11 @@ export const name = 'seam-climate-setting-schedule-table'
 
 export const props: ElementProps<ClimateSettingScheduleTableProps> = {
   deviceId: 'string',
+  disableSearch: 'boolean',
+  climateSettingScheduleFilter: 'object',
+  climateSettingScheduleComparator: 'object',
   onBack: 'object',
+  heading: 'string',
```
Comment (Andrewjeska) [inline src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleTable.element.ts:13]: I saw we use heading now, seemed small enough to stick in here

**Comment by Andrewjeska** [comment]
I think the diff we see here is just the fact that the story was renamed

**Comment by dawnho** [comment]
@Andrewjeska could you include some screenshots / videos?

File: src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleTable.tsx [hunk 3]
```diff
@@ -9,28 +12,67 @@ import { EmptyPlaceholder } from 'lib/ui/Table/EmptyPlaceholder.js'
...
+import { SearchTextField } from 'lib/ui/TextField/SearchTextField.js'
 import { Caption } from 'lib/ui/typography/Caption.js'
 
 import { ClimateSettingScheduleRow } from './ClimateSettingScheduleRow.js'
 
 export interface ClimateSettingScheduleTableProps {
   deviceId: string
+  disableSearch?: boolean
+  climateSettingScheduleFilter?: (
+    climateSettingSchedule: ClimateSettingSchedule,
```
Comment (dawnho) [inline src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleTable.tsx:24]: cc @mikewuu @razor-x, wanted to make sure we're using this property consistently

**Review by dawnho — APPROVED** [review]
Search looks good. Let's have the rest of the team review the props before merging.

**Comment by Andrewjeska** [comment]
> @Andrewjeska could you include some screenshots / videos?

I can, but we also have chromatic for this

https://www.chromatic.com/component?appId=6441abde27a258c5292861cb&csfId=example-climatesettingscheduletable-coming-soon&specName=Content&buildNumber=1365&k=64e4085d142b5b23b71d6ecf-1200px-interactive-true&h=3&b=-1

And also the preview deployment

https://react-git-climate-setting-schedules-search-filter-seamapi.vercel.app/?path=/docs/example-climatesettingscheduletable-coming-soon--docs

**Review by razor-x — APPROVED** [review]
👌

**Pull request merged**